Update "How authorization works in App Service" section

loosely-coupled opened the pull request:
The browser receiving a request does not make sense in this paragraph. Changing "receives" to "sends" is what I believe is correct.

azurecla commented:
Hi __@estei__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

Commit:
Update "How authorization works in App Service" section

The browser receiving a request does not make sense in this paragraph. Changing "receives" to "sends" is what I believe is correct.
